adapters/consumable/consumable.go

```diff
@@ -269,7 +269,7 @@ func (a *ConsumableAdapter) MakeBids(
 
 			bidderResponse.Bids = append(bidderResponse.Bids, &adapters.TypedBid{
 				Bid:     &bid,
-				BidType: getMediaTypeForImp(getImp(bid.ImpID, internalRequest.Imp)),
+				BidType: openrtb_ext.BidTypeBanner,
```

Comment:
@djcsdy Just a suggestion: Would be good to add a comment here explaining the reason why it's always set to banner.

Reply:
Done, thanks.